lib/screens/vehicles/parser.ex

```diff
@@ -19,13 +20,28 @@ defmodule Screens.Vehicles.Parser do
     %Screens.Vehicles.Vehicle{
       id: vehicle_id,
       direction_id: direction_id,
+      carriages: parse_carriages(carriages),
       current_status: parse_current_status(current_status),
       occupancy_status: parse_occupancy_status(occupancy_status),
       trip_id: trip_id_from_trip_data(trip_data),
       stop_id: stop_id_from_stop_data(stop_data)
     }
   end
 
+  defp parse_carriages(data), do: Enum.map(data, &parse_car_crowding/1)
+
+  defp parse_car_crowding(%{
+         "occupancy_status" => occupancy_status,
+         "occupancy_percentage" => occupancy_percentage,
+         "carriage_sequence" => carriage_sequence
+       }) do
+    %{
+      occupancy_status: parse_occupancy_status(occupancy_status),
```

Comment:
I believe OL crowding uses a different occupancy level system than bus, so we may need to add a `parse_occupancy_status` for just OL. If you look [here](https://www.notion.so/mbta-downtown-crossing/Orange-Line-Car-Crowding-UX-Design-7e31f94cb0ab47eeacd3489877a040d4?pvs=4#abb0744aafe1471e960d34f3d7049ea2), the API uses 5 levels and Betsy created this table for converting the 5 down to 3.